### PR TITLE
Added policy - block-cluster-admin-from-ns.yaml

### PR DESCRIPTION
**Pull request by tommeramber**
## Related Issue(s)
[x]

## Description
In this policy, I’ve told `Kyverno` to look for all user operations (`CREATE, UPDATE, DELETE`), on every object kind, in the testnamespace namespace, and for the `clusterRole cluster-admin`. I’ve also added the `subject User testuser` so it won’t include all the cluster-admins in the cluster.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Thanks for this! In addition to comments, couple things we need:

1.) Annotations. All Kyverno policies must have a full set of annotations. See the wiki here for details.
2.) Artifact Hub. Policies need the AH metadata file. See https://github.com/kyverno/policies#artifact-hub.

File: other/block-cluster-admin-from-ns/block-cluster-admin-from-ns.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: block-cluster-admin-from-ns
+  annotations:
+    policies.kyverno.io/title: Block Cluster Admin from Namespace
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/subject: RBAC
+spec:
+  validationFailureAction: enforce
```
[the remark this answers is not in the document]
Comment (tommeramber) [reply other/block-cluster-admin-from-ns/block-cluster-admin-from-ns.yaml]: Done!

**Review by tommeramber — COMMENTED** [review]
Please notify me if anything else is required

**Comment by chipzoller** [comment]
Looks like DCO sign-off is still missing.

**Comment by chipzoller** [comment]
@tommeramber DCO sign-off is still required here.

**Comment by tommeramber** [comment]
@chipzoller Done :)

**Review by chipzoller — APPROVED** [review]

**Pull request merged**